Sources/Tracking/Util/Log.swift

```diff
@@ -65,12 +67,15 @@ public class ConsoleLogger: Logger {
     private func printMessage(_ message: String, _ level: OSLogType) {
         if !minLogLevel.shouldLog(level) { return }
 
+        let logsPrefix = "(siteid:\(siteId.abbreviatedSiteId))"
```

Comment:
just to avoid too much visual clutter?

Reply:
What is to avoid too much visual clutter? Sorry, I don't understand your question.

Reply:
I think Stephen wants to ask you that did you use the variable `logsPrefix` to simplify the next line i.e.
`let messageToPrint = "\(logsPrefix) \(message)"` or what was the intention to have a separate variable for this.

Reply:
Ah! 

I think I created this variable during development to fix some issue I was having with getting the code to compile. It isn't needed anymore. I'll remove it as I agree it doesn't add any value here.